__tests__/components/editor/GroupChoiceModal.test.js

```diff
@@ -52,24 +50,28 @@ describe('<GroupChoiceModal />', () => {
       expect(wrapper.find(Modal.Body).find('form').find('select').length).toEqual(1)
     })
 
-    it('has the first select option as "PCC"', () => {
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .first()
-        .childAt(0)
-        .text()).toEqual('PCC')
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .first()
-        .prop('value')).toEqual('pcc')
+    it('has the first select option as "Cornell University"', () => {
+      const firstOpt = wrapper.find(Modal.Body).find('form select option').first()
+      expect(firstOpt.text()).toEqual('Cornell University')
+      expect(firstOpt.prop('value')).toEqual('cornell')
     })
 
-    it('has the last select option as "University of Washington"', () => {
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .last()
-        .childAt(0)
-        .text()).toEqual('University of Washington')
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .last()
-        .prop('value')).toEqual('wau')
+    it('has the last select option as "Yale University"', () => {
```

Comment:
quick question I wouldn't hold this up for -- would we want to have a test for the number of expected groups??

Reply:
if the test that @jcoyne doesn't like sticks around, i think that's unnecessary, because it is implicitly tested there (since it tests for equality against the entire expected list).

but since no one really likes that test a lot as is, if it gets reworked or discarded, maybe? 🙂